src/app/components/FilterPopup/FilterPopup.jsx

```diff
@@ -284,8 +357,21 @@ class FilterPopup extends React.Component {
             active={showForm}
             id="filter-popup-menu"
             role="menu"
-            className={`${js ? 'popup' : 'popup-no-js'} nypl-popup-filter-menu ${showForm ? 'active' : ''}`}
+            className={
+              `${js ? 'popup' : 'popup-no-js'} nypl-popup-filter-menu ${showForm ? 'active' : ''}`
+            }
           >
+            {
```

Comment:
Make this a variable instead of inline html since it's a lot.

Reply:
Do you mean this div? `<div className="nypl-form-error" ref="nypl-filter-error" tabIndex="0">`?